packages/clis/parser/game-files/map-files-parser.ts

```diff
@@ -925,6 +925,25 @@ function postProcess(
     'roads possibly split by terrains, buildings, or curves',
   );
 
+  // Augment mileage targets from defs with position info from sectors.
+  for (const [token, target] of defData.mileageTargets) {
+    if ((target.easting && target.southing) || !target.nodeUid) {
+      continue;
+    }
+    const nodeUid = BigInt(target.nodeUid);
+    const node = nodesByUid.get(nodeUid);
+    if (node) {
+      defData.mileageTargets.set(token, {
+        ...target,
+        easting: Math.round(node.x * 100) / 100,
+        southing: Math.round(node.y * 100) / 100,
+      });
+      logger.trace('node', nodeUid, 'found for mileage target', token);
+    } else {
+      logger.debug('node', nodeUid, 'not found for mileage target', token);
```

Comment:
Maybe. Does a regular user want to know this? I can't really say.

The argument **for** would be that some users might be able to fix these missing positions on their end. For example, a few of the missing node UIDs did exist in earlier game versions. If you have a list of node UIDs and their positions from like 1.49, you can use it to fix some of these targets. (Though I suspect there aren't many users who would be able to do this.)

The argument **against** is that it clutters up the log. Users who don't have every DLC are particularly affected. If you run this without `includeDlc`, you'll get like 700 lines of "node not found" entries that may look scary, but are ultimately meaningless if you don't own the DLC anyway.

I'd tend towards keeping it as `debug`.

By the way, I found it easy to set the logging level with an environment variable. I don't think a `parser` option is necessary, but perhaps this might be mentioned in the readme.
```sh
CONSOLA_LEVEL=4 npx parser ...
```

Reply:
> By the way, I found it easy to set the logging level with an environment variable. I don't think a `parser` option is necessary, but perhaps this might be mentioned in the readme.

🤯 I did not know about the `CONSOLA_LEVEL` env var... and is why I've been overly-favoring warnings; I am almost always in "debug" mode, myself.

I think you're absolutely right when you say:

> My rule of thumb for debug vs. info or higher is: "Does a regular user want to know this?"

and I will re-audit the `logger.warn`s in use 😅

Reply:
I mean, you *are* the primary user of this software, right? I think it's fair to optimize for that. A re-audit isn't super urgent in my view.

For what it's worth, I found the env var [somewhere deep down in the Consola readme](https://github.com/unjs/consola/blob/faa9cbd29d0fe89c5919d20c81c64660a04eb6ac/README.md#log-level).